lib/Smokeping.pm

```diff
@@ -1372,7 +1372,7 @@ sub get_detail ($$$$;$){
             my @lazy =();
             @lazy = ('--lazy') if $mode eq 's' and $lastheight{$s} and $lastheight{$s}{$start} and $lastheight{$s}{$start} == $max->{$s}{$start};
             my $timer_start = time();
-            my $from = $s ? " from $cfg->{Slaves}{$slave}{display_name}": "";
+            my $from = " from " . ($s ? $cfg->{Slaves}{$slave}{display_name}: $cfg->{General}{display_name} || hostname);
```

Comment:
does `||` bind more tightly than the ternary operator ? also since the host name is not bound to change, it might make sense to use the hostname as a default for display_name if that is not already the case.